src/hal/lv_hal_disp.h

```diff
@@ -77,8 +77,13 @@ typedef enum {
  */
 typedef struct _lv_disp_drv_t {
 
-    lv_coord_t hor_res;         /**< Horizontal resolution.*/
-    lv_coord_t ver_res;         /**< Vertical resolution.*/
+    lv_coord_t hor_res;         /**< Horizontal resolution of the display portion that is being rendered to.*/
+    lv_coord_t ver_res;         /**< Vertical resolution of the display portion that is being rendered to.*/
+
+    lv_coord_t display_hor_res;      /**< Horizontal resolution of the full display. Set to -1 for fullscreen mode.*/
```

Comment:
I'd prefer using standard terms if possible (e.g. I think this should really be called `stride`, not sure about `display_ver_res` though), as otherwise these will be confused with `hor_res` and `ver_res`.

Reply:
`stride` sounds reasonable at fist look but thinking more about it it's also confusing. Let's say LVGL has 320 px hor_res to draw. The display has 400 px hor res. Bur for optimal memory access the the frame buffer has N x 32 px hor res = 416 px. In this case the "real" stride is 416. And 400 is only the "real" resolution of the display.  

However, if LVGL works in `disp_drv->full_refresh = 1` and the stride is different from the LVGL's hor res, LVGL will render incorrectly as LVGL can't handle stride now. (But it should.) 

So it seems to me `stride` is an new parameter on top of `hor/ver_res` and `display_hor/ver_res`. 

`display_hor_res` is really a little bit unfortunate name because it's easy to mix with `hor_res`. Something like `physical_hor_res` or `real_hor_res`?

Reply:
Agree that the naming gets confusing with two displays. :disappointed: 

I changed it to `physical_hor_res` / `physical_ver_res` and `offset_x` / `offset_y` and also updated the comments / docs a little to make the meaning more clear. Let me know if that seems ok.